row_025_views/row_025_views_safe.py

```
@@ -0,0 +1,778 @@
+    # Display a success message
+    messages.success(request, _("Item has been unshared successfully."))
+    
+    # Redirect back to the item view page
+    return redirect('view_item', item_uuid=item.id)
+
+# API
+
+@require_authorization_header_with_api_token
+
+def get_items_by_type(request, item_type):
+    authenticate_general_api_key(request)
```

Ruff (0.14.2)

`authenticate_general_api_key` may be undefined, or defined from star imports

(F405)